README.md

```diff
@@ -54,9 +54,9 @@ By default, the plugin does not hide any accidental printing of secret to consol
 pipeline {
     agent any
     environment {
-        SECRET1    = vault path: 'secrets', key: 'password1', vaultUrl: 'https://my-vault.com:8200', credentialsId: 'my-creds'
-        SECRET2    = vault path: 'secrets', key: 'password2', vaultUrl: 'https://my-vault.com:8200', credentialsId: 'my-creds'
-        NOT_SECRET = vault path: 'secrets', key: 'username', vaultUrl: 'https://my-vault.com:8200', credentialsId: 'my-creds'
+        SECRET1    = vault path: 'secrets', key: 'password1', vaultUrl: 'https://my-vault.com:8200', credentialsId: 'my-creds', engineVersion: 2
```

Comment:
Does not have to be mandatory.

The default should be publicly available on `VaultConfiguration.DescriptorImpl.DEFAULT_ENGINE_VERSION`